.ci/containers/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -240,7 +277,7 @@ if [[ -n $FAILED_TESTS_PATTERN ]]; then
       # check for any uncaught errors in RECORDING mode
       comment+="Errors occurred during RECORDING mode. Please fix them to complete your PR${NEWLINE}"
     else
-      comment+="All tests passed${NEWLINE}"
+      comment+="All tests passed during RECORDING mode.${NEWLINE}"
```

Comment:
Should we remove this? Will this comment be confusing to contributors if tests passed in recording but failed in replaying? Can you think of a case that we still need this line?

Reply:
Good catch. The else block is not needed. I will remove these lines.